syncthing.fc

```diff
@@ -0,0 +1,8 @@
+# binary
+/usr/bin/syncthing                      -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+/usr/local/bin/syncthing                -- gen_context(system_u:object_r:syncthing_exec_t,s0)
+HOME_DIR/.local/bin/syncthing           -- gen_context(system_u:object_r:syncthing_exec_t,s0)
```

Comment:
On 08/21/2016 09:14 PM, Naftuli Tzvi Kay wrote:
>> @@ -0,0 +1,8 @@
>> +# binary
>> +/usr/bin/syncthing                      -- gen_context(system_u:object_r:syncthing_exec_t,s0)
>> +/usr/local/bin/syncthing                -- gen_context(system_u:object_r:syncthing_exec_t,s0)
>> +HOME_DIR/.local/bin/syncthing           -- gen_context(system_u:object_r:syncthing_exec_t,s0)
> 
> There are multiple ways of installing syncthing. I could probably condense `/usr/(local/)?bin/syncthing`.
> 
> Installing Syncthing as a fat binary in your homedir allows Syncthing to automatically update itself outside of package management. While that's a whole different problem, some users might want this for distros that aren't yet packaging it (ie Fedora).
> 

Naw. that would never work this way. Users will not be able to manage
that file in ~/.local/bin. Just drop that.

-- 
Key fingerprint = 5F4D 3CDB D3F8 3652 FBD8  02D5 3B6C 5F1D 2C7B 6B02
https://sks-keyservers.net/pks/lookup?op=get&search=0x3B6C5F1D2C7B6B02
Dominick Grift